test/e2e/kcp_remediation_test.go

```diff
@@ -23,16 +23,25 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	"k8s.io/utils/ptr"
 	capi_e2e "sigs.k8s.io/cluster-api/test/e2e"
+	"sigs.k8s.io/cluster-api/test/framework/clusterctl"
 )
 
 var _ = Describe("When testing KCP remediation", func() {
+	if clusterctl.DefaultInfrastructureProvider == "aws" {
+		// Skip the test for cloud provider as it is not supported
+		// See kubernetes.slack.com/archives/C8TSNPY4T/p1680525266510109
+		// And github.com/kubernetes-sigs/cluster-api-provider-aws/issues/4198
+		return
```

Comment:
Ginko also allows to explicitly skip tests (`ginko.Skip()` or something like this)